### PR TITLE
Reintroduce CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by jablko**
The "How can I contribute?" section of the README is quite long, would it make sense to [reintroduce](https://github.com/DefinitelyTyped/DefinitelyTyped/pull/14822) a [CONTRIBUTING file](https://docs.github.com/en/free-pro-team@latest/github/building-a-strong-community/setting-guidelines-for-repository-contributors)?

### CODE REVIEW

**Review by elibarzilay — CHANGES_REQUESTED** [review]
See the comment about changing the text.  It also made some good changes, but in general I think that it's best to keep it to two separate PRs: one that moves the text as-is (which is probably just removing the second commit here), and another that does the markdown tweaking (but the way that it did it here, I have many objections...).

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -3,8 +3,8 @@ Please fill in this template.
 - [ ] Use a meaningful title for the pull request. Include the name of the package modified.
 - [ ] Test the change in your own code. (Compile and run.)
 - [ ] Add or edit tests to reflect the change. (Run with `npm test YOUR_PACKAGE_NAME`.)
-- [ ] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
-- [ ] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
+- [ ] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/CONTRIBUTING.md#make-a-pull-request).
```
[the remark this answers is not in the document]
Comment (jablko) [reply .github/PULL_REQUEST_TEMPLATE.md]: Done. :heavy_check_mark:

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,477 @@
+# How can I contribute?
```
[the remark this answers is not in the document]
Comment (jablko) [reply CONTRIBUTING.md:1]: Yup, unedited. I've updated the log message. :heavy_check_mark:

**Comment by jablko** [comment]
@elibarzilay Thanks for the feedback, I've separated the commits. :heavy_check_mark:

**Review by sandersn — CHANGES_REQUESTED** [review]
I don't like having a separate file, since (1) the majority of people reading Definitely Typed's README are there to contribute (2) non-contributors can stop reading early on, since the useful command `npm i @types/x` happens before any contributing instructions.

Since contributing is the default case, and since github puts README on the front page of a repo, it makes sense to include the contributing instructions, even though they're long.

Thoughts from other DT maintainers?

**Comment by orta** [comment]
Yeah, I think I'm with Nathan here - the README is basically only the contributors guideline. There's a part of me that's wanted to move some of this to the site, but the co-location is super valuable and I've punted on it also for the same reasons.

**Comment by jablko** [comment]
Or, are there other changes I could make to aid navigation? No disagreement that it's basically the contributors guideline, but I find some details get buried in there.

**Comment by elibarzilay** [comment]
Maybe make a new contributing.md which has a one-page command-line-centric summary of how to make a PR, with a pointer to the readme for further details?

**Comment by elibarzilay** [comment]
(Closing this since it looks like it not a popular change...)

**Pull request closed without merging**